### DISCUSSION TITLE
Are there dynamic variables for download path and auto-naming of files?

### DISCUSSION THREAD

**Discussion by jardenz** [aclap-dev/video-downloadhelper#405] [Help and Q&A]
### Your question

I've looked through the VDH docs but unable to find any information on whether any dynamic variables are supported for the default download directory, or auto-naming of files. 

Example, I'd like the default download path to be dynamic, and save to a folder named after today's date (and create folder if it doesn't exist), i.e.

**D:\DL\\%d-%m-%Y  -->  D:\DL\04-16-2024**

Or being able to auto-name files based on a predefined syntax, i.e. 

**%sitename.%y.%m.%d.%title.%resolution.%ext  -->  Youtube.24.04.16.Title_of_video.2160p.mp4**

etc

### Addon & CoApp version and details

```

version: 8.2.2.9
channel: stable
build date: 2024-03-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: en-US
license: accepted
key (only 16 first characters): XXXXXXXXXXXXXXXXX
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper App\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\User",
  "converterBinary": "C:\\Program Files\\DownloadHelper App\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

```

**Comment by jcfrog** [comment]
Hi,
No, sorry, it is not a dynamic field.
